include/eve/traits/overload/protocol.hpp

```diff
@@ -169,6 +169,7 @@ inline constexpr auto adl_helper = adl_helper_t {}
 //! @}
 //======================================================================================================================
 #define EVE_REQUIRES(ARCH) adl_helper_t const &, ARCH const &
+#define EVE_TARGETS(ARCH)   adl_helper_t{}, ARCH{}
```

Comment:
is this retarget?

Reply:
It is yes. It is not called RETARGET cause it would have conflicted with the old one

Reply:
How to call it? `abs_(EVE_TARGETS(cpu_), options, args)`?

Reply:
Yes:
`return abs_(EVE_TARGETS(cpu_), opts, v);`